src/main/java/com/mongodb/jdbc/MongoDatabaseMetaData.java

```diff
@@ -833,73 +833,6 @@ protected String getDataTypeBytesCase(String col) {
         return getTypeCase(col, typeBytes);
     }
 
-    protected MongoResultDoc getTypeInfoMetaDoc() {
```

Comment:
These should have been moved into the `MySQLDatabaseMetaData` class before, we missed these.